addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -26,16 +26,13 @@
                         <a t-if="phone" class="o-mail-avatar-card-tel text-truncate" t-att-title="phone" t-att-href="'tel:'+phone">
                             <i class="fa fa-fw fa-phone me-1"/><t t-esc="phone"/>
                         </a>
-                    </div>
-                </div>
-                <div class="flex-wrap gap-2 mt-2">
-                    <div class="justify-content-end d-flex align-items-baseline">
-                        <div class="d-flex gap-2 o_avatar_card_buttons">
+                        <div class="o_avatar_card_buttons d-flex gap-2 mt-2">
                             <button t-if="!user.share" class="btn btn-secondary btn-sm" t-on-click.stop="onSendClick">Send message</button>
                             <button t-if="showViewProfileBtn" class="btn btn-secondary btn-sm" t-on-click.stop="onClickViewProfile" t-ref="viewProfileBtn">View Profile</button>
                         </div>
                     </div>
                 </div>
+                <div t-if="skills?.length or roles?.length" class="o_avatar_card_details d-flex flex-column gap-3 mx-n3 mb-n3 mt-3 border-top rounded-bottom p-3"/>
```

Comment:
IMO you can move this div outside of the `.card-body` so you don't have to use the negative margins.

That said, if we want to go a bit further here: I think it's weird to have an `.card-body` without a parent `.card`. We could add `card rounded border-0` to `.o_avatar_card` and change the classes of `.o_avatar_card_details` to `o_avatar_card_details card-footer d-flex flex-column gap-3 rounded-bottom bg-100` 

This would be semantically more correct and would fix the bg-color issue in dark mode and you wouldn't have to use the SCSS hack that you made.

Bonus point, the buttons would be more visible as well.

| This PR | Suggestion |
|--------|--------|
| <img width="401" alt="Screenshot 2024-12-02 at 17 54 00" src="https://github.com/user-attachments/assets/6a85cbe4-8874-40b6-a9b0-d4ca134f4615"> | <img width="401" alt="Screenshot 2024-12-02 at 17 55 11" src="https://github.com/user-attachments/assets/2b81d343-1547-4161-8492-0c51e36b0467"> |